docs/tutorials.md

```diff
@@ -0,0 +1,199 @@
+# Tutorials
+This explanatory tutorial will explain different keywords, terms, and rules from the perspective of YAML format of the NeXus schema. Therefore it provides a overall grasp on how to write a NeXus schema those syntactical components.
```

Comment:
Paragraph needs to be completely rewritten.

On the one hand for grammar and language.
On the other hand for precision of the formulations,
a NeXus data schema's XML schema may have a serialization in YAML but other than this there is no connection between NeXus and YAML.